edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/agentStart.sh

```diff
@@ -17,33 +17,20 @@
 #
 # This preserves backwards compatibility with earlier versions of edgeAgent and
 # allows some flexibility in the assignment of the edgeagent user id. The default 
-# is UID 1000.
+# is UID 13622.
 #
 # A user is created because at this time DotNet Core 2.x and 3.x can only install
 # trust bundles into system stores or user stores.  We choose a user store in
 # the code, so a writeable user directory is required.
 ###############################################################################
 echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Starting Edge Agent"
 
-TARGET_UID="${EDGEAGENTUSER_ID:-1000}"
+TARGET_UID="${EDGEAGENTUSER_ID:-13622}"
 cuid=$(id -u)
 
 if [ $cuid -eq 0 ]
 then
 
```

Comment:
The adduser was placed here if a user decides to use a EDGEAGENTUSER_ID!=13622 in the module spec. Can we confirm we don't need it?

Reply:
You're right, fixed.